AdyenCard/Utilities/CardPublicKeyProvider/CardPublicKeyProvider.swift

```diff
@@ -31,60 +31,43 @@ internal final class CardPublicKeyProvider: AnyCardPublicKeyProvider {
                   apiClient: AnyRetryAPIClient? = nil) {
         self.apiContext = apiContext
         if let apiClient = apiClient {
```

Comment:
How about generate `UniqueAssetAPIClient<ClientKeyResponse>` right on spot ?

Reply:
its verbose, I like the lazy var because its cleaner

Reply:
I changed it in both `CardPublicKeyProvider` and `AppleWalletPassProvider`.